src/test/groovy/com/benjaminsproule/swagger/gradleplugin/SpringMvcSampleITest.groovy

```diff
@@ -0,0 +1,70 @@
+package com.benjaminsproule.swagger.gradleplugin
+
+import groovy.json.JsonSlurper
+
+import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
+
+class SpringMvcSampleITest extends AbstractPluginITest {
+
+    def 'Configure global security'() {
+      given:
+      def expectedSwaggerDirectory = "${testProjectOutputDir}/swaggerui-" + UUID.randomUUID()
+      buildFile << """
+            plugins {
+                id 'java'
+                id 'groovy'
+                id 'com.benjaminsproule.swagger'
+            }
+            swagger {
+                apiSource {
+                    springmvc = true
+                    locations = ['com.benjaminsproule.swagger.gradleplugin.test.springmvc.SampleController']
+                    host = 'localhost:8080'
+                    basePath = '/'
+                    info {
+                        title = 'test'
+                        version = '1'
+                    }
+                    swaggerDirectory = '${expectedSwaggerDirectory}'
+                    ${testSpecificConfig}
+                }
+            }
+        """
+
+      when:
+      def result = runPluginTask()
+
+      then:
+      result.task(":${GenerateSwaggerDocsTask.TASK_NAME}").outcome == SUCCESS
+
+      assertSwaggerJsonGlobalSecurity("${expectedSwaggerDirectory}/swagger.json")
+
+      where:
+      testSpecificConfig << [
```

Comment:
Might as well have this all together with the other part of the generated file. Or create another one to prove multiple differing values work properly?

Also, I know this isn't Spring/JAX-RS specific, but it would be nice to have a JAX-RS test as well, just to make sure it's not broken in a future change.

Reply:
Will add additional tests